mitodl/ocw-studio
# Remove and migrate dependency ckeditor5-dev-webpack-plugin

### What are the relevant tickets?
No tickets
Dependency Dashboard: https://github.com/mitodl/ocw-studio/issues/1787

### Description (What does it do?)
`@ckeditor/ckeditor5-dev-webpack-plugin` is deprecated dependency. This PR migrates to using `@ckeditor/ckeditor5-dev-translations` over `@ckeditor/ckeditor5-dev-webpack-plugin`.

### How can this be tested?
1. Checkout to this branch
2. Spin up ocw studio
3. Make sure Ckeditor functionalities / build process works as expected.
